modules/modules.nix

```
@@ -124,6 +124,7 @@ let
     (loadModule ./programs/taskwarrior.nix { })
     (loadModule ./programs/termite.nix { })
     (loadModule ./programs/texlive.nix { })
+    (loadModule ./programs/thunderbird.nix { })
```

```suggestion
    (loadModule ./programs/thunderbird.nix { condition = hostPlatform.isLinux; })
```

Thunderbird from Nixpkgs doesn't support Darwin.

Unless there is an alternative package that provides Thunderbird for Darwin on Nixpkgs, this will need to be gated.
(And if it exists, you should modify the logic for `default` value of the `package` options.)

---

Do the tests also have to be gated?

---

Yup